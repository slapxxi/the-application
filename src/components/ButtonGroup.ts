import styled from 'styled-components';
import Button from './Button';

const ButtonGroup = styled.div`
  ${Button} {
    border-radius: 0;
    margin: 0;
  }

  ${Button}:first-child {
    border-radius: 4px 0 0 4px;
  }

  ${Button}:last-child {
    border-radius: 0 4px 4px 0;
  }
`;

export default ButtonGroup;
