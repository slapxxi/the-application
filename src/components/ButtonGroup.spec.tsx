import React from 'react';
import ButtonGroup from './ButtonGroup';
import renderer from 'react-test-renderer';

it('renders', () => {
  const component = renderer
    .create(<ButtonGroup>hello</ButtonGroup>)
    .toJSON();
  expect(component).toMatchSnapshot();
});
