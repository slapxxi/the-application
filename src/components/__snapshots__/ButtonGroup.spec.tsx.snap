// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 .sc-bdVaJa {
  border-radius: 0;
  margin: 0;
}

.c0 .sc-bdVaJa:first-child {
  border-radius: 4px 0 0 4px;
}

.c0 .sc-bdVaJa:last-child {
  border-radius: 0 4px 4px 0;
}

<div
  className="c0"
>
  hello
</div>
`;
